sim/paladin/items.go

```diff
@@ -109,3 +181,67 @@ func init() {
 		})
 	})
 }
+
+func blazefuryTriggerAura(character *core.Character, spellID int32, spellSchool core.SpellSchool, damage int32) *core.Aura {
```

Comment:
Did you want to share this aura with Blazefury Medallion or just use it for the paladin weapons?

Reply:
I can share it